build/common/utils.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 function init_conf {
-    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
@atb00ker is this really needed? I have noticed that also without this the timezone is set correctly in the containers (checked using the `date` command).

Reply:
The logs might display time in UTC without this command because, I think supervisord relies on the file in `/etc/localtime` to get the timezone for logs.

But if all the logs show correct time without this command now, I am good with removing it! 😄

Reply:
The date command returns the right date in every container. Is there anything else I should try? That should be enough, if the date command returns the right date all the rest should follow since that's the system time.